CHANGELOG.rst

```diff
@@ -4,6 +4,14 @@ Change Log
 ..
    This file loosely adheres to the structure of https://keepachangelog.com/,
    but in reStructuredText instead of Markdown.
+2020-11-06
+----------
+
+Changed
+~~~~~~~
+
+* Rename ``doc.{in,txt}`` to ``docs.{in,txt}`` to reduce work when setting up
+  Read the Docs.
```

Comment:
May be worth noting the RTD default thing that's mentioned in the commit message.

Reply:
Turns out it wasn't an RTD default but one we were incorrectly setting in our `.readthedocs.yml` config file.  Updated to be a much smaller change that fixes the typo.